### PR TITLE
MSM comments to the issue 40 code

### PR DESCRIPTION
**Pull request by mitchute**
These are a few minor changes which should be considered before merging the pipe networks work into the master branch. Most of the auto formatting by PyCharm has been omitted, however, there were a few unused imports so they have been formatted automatically by PyCharm. The major areas of interest will be called out in comments.

### CODE REVIEW

File: pygfunction/pipes.py [hunk 1]
```diff
@@ -370,7 +368,7 @@ def coefficients_temperature(self, z, m_flow, cp, nSegments):
 
         """
         # method_id for coefficients_temperature is 5
-        method_id = 5
+        # method_id = 5  Unused: check
```
Comment (mitchute) [inline pygfunction/pipes.py]: ```method_id``` is not used here. Check usage.

File: pygfunction/pipes.py [hunk 2]
```diff
@@ -1156,6 +1215,8 @@ def _continuity_condition_base(self, m_flow, cp, nSegments):
             a_in = np.linalg.multi_dot([e_u, d_u_m1, d_in])
             a_out = np.array([[1.0]])
             a_b = np.linalg.multi_dot([e_u, d_u_m1, d_b])
+        else:
```
Comment (mitchute) [inline pygfunction/pipes.py]: Fallthrough catch needed.

File: pygfunction/pipes.py [hunk 3]
```diff
@@ -1156,6 +1215,8 @@ def _continuity_condition_base(self, m_flow, cp, nSegments):
...
+            raise NotImplementedError("Configuration '{}' not implemented.".format(self.config))
```
Comment (mitchute) [inline pygfunction/pipes.py]: Fallthrough catch needed.

File: pygfunction/pipes.py [hunk 4]
```diff
@@ -1231,6 +1292,8 @@ def _continuity_condition_head(self, m_flow, cp, nSegments):
             a_out = np.zeros((2*self.nPipes, self.nOutlets))
             a_b = np.linalg.multi_dot([e_d, c_u, d_u_m1, d_b]) \
                 + np.linalg.multi_dot([e_u, d_u_m1, d_b])
+        else:
```
Comment (mitchute) [inline pygfunction/pipes.py]: Fallthrough catch needed.

File: pygfunction/heat_transfer.py [hunk 5]
```diff
@@ -295,12 +291,14 @@ def similarities(boreholes, splitRealAndImage=True, disTol=0.1, tol=1.0e-6,
         Set to True if similarities are evaluated separately for real and image
         sources. Set to False if similarities are evaluated for the sum of the
         real and image sources.
-    distol : float, defaults to 0.1
+    disTol : float, defaults to 0.1
         Absolute tolerance (in meters) on radial distance. Two distances
         (d1, d2) between two pairs of boreholes are considered equal if the
         difference between the two distances (abs(d1-d2)) is below tolerance.
+
+    # tol is not used
```
Comment (mitchute) [inline pygfunction/heat_transfer.py]: This isn't passed through to anything.

File: pygfunction/pipes.py (hunk 1 above)
Comment (MassimoCimmino) [reply pygfunction/pipes.py]: This one should probably stay, as there may be a need for memory in the future.

File: pygfunction/pipes.py (hunk 2 above)
Comment (MassimoCimmino) [reply pygfunction/pipes.py]: This is good, thanks.

File: pygfunction/pipes.py (hunk 4 above)
Comment (MassimoCimmino) [reply pygfunction/pipes.py]: This is good, thanks.

File: pygfunction/heat_transfer.py (hunk 5 above)
Comment (MassimoCimmino) [reply pygfunction/heat_transfer.py]: I've found the missing argument in calls to `_similarities_one_distance`.

**Comment by MassimoCimmino** [comment]
Thanks for all the work @mitchute. I will pull this into #40 and work on a few changes you proposed.

**Pull request merged**